civitai-downloader-v2/src/core/adaptability/api_detector.py

```
@@ -0,0 +1,380 @@
+        Args:
+            force_full_scan: Force a complete API scan
+            
+        Returns:
+            List of detected changes
+        """
+        logger.info("Starting API change detection")
+        changes = []
+        
+        try:
+            # Scan all known endpoints
+            for endpoint_key, endpoint in self.endpoints.items():
```

Ruff (0.12.2)

Loop control variable `endpoint_key` not used within loop body

Rename unused `endpoint_key` to `_endpoint_key`

(B007)